specification/azsadmin/resource-manager/containerservice/Microsoft.ContainerService.Admin/stable/2019-11-01/managedClusters.json

```diff
@@ -0,0 +1,153 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01",
+    "title": "ContainerServiceAdminClient",
+    "description": "Admin Container Service Management Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerService.Admin/locations/{location}/managedClusters": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of managed clusters at the given location.": {
+            "$ref": "./examples/managedClusters/List.json"
+          }
+        },
+        "tags": [
+          "ContainerServices"
+        ],
+        "description": "Returns a list of managed clusters present in all tenant location.",
+        "operationId": "ContainerServices_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of managed clusters has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerServicesList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerServiceProperties": {
+      "description": "Container service properties.",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
id, name, location, and subscriptionId are either top level properties that must not be repeated inside the property bag or implied by the URL in subscriptionIDs case.  Please remove them from the property bag.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-request-properties-envelope

Reply:
ID, subscriptionId, location under properties are Azure Stack user namespace kubernetes cluster details. These are different from Azure Stack Admin namespace id, subscriptionId and location top level entities